spec/requests/ebooks_spec.rb

```diff
@@ -62,13 +62,24 @@
             )
           end
           let(:parent) { instance_double(Sighrax::Entity, title: 'title') }
+          let(:entity_presenter) { double("entity_presenter") }
+          let(:counter_service) { double("counter_service") }
 
           before do
             allow(entity).to receive(:content).and_return(File.read(Rails.root.join(fixture_path, entity.filename)))
+            allow(Sighrax).to receive(:hyrax_presenter).with(entity).and_return(entity_presenter)
+            allow(CounterService).to receive(:from).and_return(counter_service)
+            allow(counter_service).to receive(:count).with(request: 1).and_return(true)
```

Comment:
Can we add the `expect(counter_service)` checks to the it block(s) as are done in e_pubs_spec.rb?